### ISSUE TITLE
A1A and accessibility

### ISSUE THREAD

**Issue by styssi** [vandeseer/easytable#48]
Hi, I am planning on adding A1A and accessibility to the easytable framework, because I need this. Can you please add a branch for me, I can work on. I would like to push some code in between, instead of pushing everything in the end.

**Comment by vandeseer** [comment]
Hi @styssi, I can do that, but I don't yet understand why it's needed: You could just create a branch on your fork (and rebase onto or merge "my" `develop` branch once in a while) and whenever you have something that should be included, you just open a pull request. From my point of view that'd be the easiest, but if there is any stronger point in open a separate branch on the original repo, we can do that as well ...

**Comment by styssi** [comment]
Ok. Is fine with me.

**Comment by vandeseer** [comment]
@styssi I refactored the code a bit in the last days on the `develop` branch (mainly the `TableDrawer` class). Just FYI ;) 
Do you need any help or are you fine with working on your own branch etc.? If so, I would close this issue ...

**Comment by styssi** [comment]
@vandeseer the issue can be closed. This is more refactoring, than I thought. If I would merge this, than easytable isn't downward compatible anymore.

**Comment by vandeseer** [comment]
Maybe you can upload your changes to a branch of your repository and we can have a look on how to integrate it in the latest version of `develop`? I don't know how extensive your changes are and what exactly they will introduce, but it would be nice to have a look for sure.

**Comment by styssi** [comment]
I can do that. Currently nothing is working :), but I will upload as soon as I have a stable release.

**Comment by phautamaki** [comment]
Hello! Any updates on this?

**Comment by styssi** [comment]
I am no longer working on this

**Comment by phautamaki** [comment]
OK. How did you solve the accessilibity requirements in your case? Still using easytable?

**Comment by styssi** [comment]
No. We did build our own implementation. Since it is for a customer we cannot make the code public. I can tell you this. It took a lot of time.

**Comment by phautamaki** [comment]
Yep. We are also in the middle of making some PDF:s more accessible and the parts outside tables are doable (I think) with pure PDFBox. However, it seems the tables created with easytable cannot be easily transformed without rewriting some of the easytable code. We'll see what we end up doing in our case. Easytable is a great library in general, and the accessibility features would make it even greater – maybe I'll give this a shot at some point.
